guide/14-deep-learning/object-detection.ipynb

```diff
@@ -431,7 +431,8 @@
    "source": [
```

Comment:
<p># only EMD inside parenthesis</p><p><em>This creates an Esri Model Definition (EMD) file&nbsp;</em></p><p> </p><p>#sentence is long, could be broken -</p><blockquote>This creates an Esri Model Definition (EMD file) that can be used for inferencing in ArcGIS Pro as well as a Deep Learning Package (DLPK zip) that can be deployed to ArcGIS Enterprise for distributed inferencing across a large geographical area using raster analytics. Saved models can also be loaded back using the&nbsp;<code>load()</code>&nbsp;method, for futher fine tuning.</blockquote><p>Suggestion:</p><p>This creates an Esri Model Definition (EMD file) that can be used for inferencing in ArcGIS Pro as well as a Deep Learning Package (DLPK zip). The DLPK package can be deployed to ArcGIS Enterprise for distributed inferencing across a large geographical area using raster analytics. </p><p> </p><blockquote><code>save()</code>&nbsp;method takes in additional argument&nbsp;<code>framework</code>&nbsp;which defaults to&nbsp;<code>PyTorch</code></blockquote><p> </p><p>#new paragraph for tensorflow part, and two sentence could be joined since there is repetition:-</p><p> </p><p>Models can be saved in " Tensorflow-onnx " by setting&nbsp;<code>framework='TF-ONNX'</code> , which can be used to perform inferencing using tensorflow or onnx libraries. This feature provides cross-platform functionality to&nbsp;<code>arcgis.learn</code>&nbsp;models, where a model trained in&nbsp;<code>PyTorch</code>&nbsp;can be used for inference in&nbsp;<code>Tensorflow</code>&nbsp;and&nbsp;<code>Onnx</code>.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>